internal/common/common.go

```diff
@@ -5,8 +5,6 @@ const (
 	AnnotationKeyGitSyncInstance = "numaplane.numaproj.io/tracking-id"
 	// SSAManager is the default numaplane manager name used by server-side apply syncs
 	SSAManager = "numaplane-controller"
-	// EnvLogFormat log format that is defined by `--logformat` option
-	EnvLogFormat = "NUMAPLANE_LOG_FORMAT"
 	// EnvLogLevel log level that is defined by `--loglevel` option
```

Comment:
nit: is this comment still hold true? e.g. we still have`--loglevel` ?

Reply:
I removed the NUMAPLANE_LOG_FORMAT because it no longer applies.